PWGDQ/Tasks/tableReader.cxx

```diff
@@ -54,12 +54,15 @@ DECLARE_SOA_COLUMN(MixingHash, mixingHash, int);
 DECLARE_SOA_COLUMN(IsEventSelected, isEventSelected, int);
 DECLARE_SOA_COLUMN(IsBarrelSelected, isBarrelSelected, int);
 DECLARE_SOA_COLUMN(IsMuonSelected, isMuonSelected, int);
+DECLARE_SOA_COLUMN(IsBarrelSelectedDalitz, isBarrelSelectedDalitz, int);
+DECLARE_SOA_COLUMN(IsDalitzLeg, isDalitzLeg, int);
```

Comment:
This is a general comment. We need to remove Dalitz from the naming. This is about prefiltering, which is mainly about removing conversions. The prefiltering is also the usual name which was used in DQ since the very beginning. Please remove Dalitz from names everywhere.